docs/en/Deployment.md

```diff
@@ -0,0 +1,145 @@
+# Operation and maintenance management
+
+## Cluster building
+
+### 1 Single Master mode
+
+This is the simplest, but also the riskiest, mode that makes the entire service unavailable once the Broker restarts or goes down.Online environments are not recommended, but can be used for local testing and development.Here are the steps to build。
+
+**1）Start NameServer**
+
+```shell
+### Start Name Server first
+$ nohup sh mqnamesrv &
+ 
+### Then verify that the Name Server starts successfully
+$ tail -f ~/logs/rocketmqlogs/namesrv.log
+The Name Server boot success...
+```
+
+We can see 'The Name Server boot success.. ' in namesrv.log That indicates that the start NameServer has been successful.
+
+**2）Start Broker**
+
+```shell
+### Also start Broker first
+$ nohup sh bin/mqbroker -n localhost:9876 &
+
+### Then verify that the Broker is started successfully, for example, the IP of Broker is 192.168.1.2 and the name is broker-a
+$ tail -f ~/logs/rocketmqlogs/Broker.log 
+The broker[broker-a, 192.169.1.2:10911] boot success...
+```
+
+We can see 'The broker[brokerName,ip:port] boot success.. ' in Broker.log That indicates that the start Broker has been successful.
+
+### 2 Multiple Master mode
+
+The advantages and disadvantages of a cluster without Slave, being all Master, such as 2 Master or 3 Master, are as follows:
```

Comment:
It will be easier to understand if we split this sentence into two，something like this:
Multiple master mode means a mode with all master nodes(such as 2 or 3 master nodes) and no slave node. The advantages and disadvantages of this mode are as follows: